### PR TITLE
Sy/translation

### PR DESCRIPTION
**Pull request by shuwnyuantee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
 # translations-websockets-api
 Language files for Binary.com Websockets API service
+
+### Install
+* apt-get install gettext
+
+### Steps to grab / update all texts to be translated from code:
+* goto bom-backoffice repo, run `make i18n`
```
Comment (raunakkathuria) [inline README.md]: do we want to leak this bom-backoffice thing?

Comment (shuwnyuantee) [inline README.md]: actually I also started to think about it, will move it to bom-backoffice

Comment (shuwnyuantee) [inline README.md]: Done move to BO

**Comment by jy-deriv** [comment]
We still leak variable names in the comments:
#. ($client->currency, $self_exclusion->max_losses)
Can those be removed too?

**Comment by shuwnyuantee** [comment]
@jy-binary  all meaningless comments are gone

**Pull request merged**